.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "third_party/D3D12MemoryAllocator"]
-	path = third_party/D3D12MemoryAllocator
-	url = https://github.com/GPUOpen-LibrariesAndSDKs/D3D12MemoryAllocator.git
```

Comment:
I suppose we can get rid of this file completely now.

Reply:
True, Ive now removed it from the repo